CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [2.0.1]
+* Update library description [#95]
```

Comment:
This isn't really what that PR does - it adds support for redirect_uris as a dictionary for RTS

Reply:
hmm. well it doesn't really add support (code-wise) as functionality is already there.  The PR updates the description of the method and adds a spec to show how this dictionary format should be provided